packages/admin/dashboard/src/routes/locations/location-service-zone-shipping-option-create/location-service-zone-shipping-option-create.tsx

```diff
@@ -37,6 +40,7 @@ export function LocationServiceZoneShippingOptionCreate() {
           zone={zone}
           isReturn={isReturn}
           locationId={location_id!}
+          type={fulfillmentSet?.type as FulfillmentSetType}
```

Comment:
suggestion: make this type safe by throwing an error if we can't find a fulfillment set with the id.